clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1556,7 +1556,7 @@ public void seekToBeginning(Collection<TopicPartition> partitions) {
         try {
             Collection<TopicPartition> parts = partitions.size() == 0 ? this.subscriptions.assignedPartitions() : partitions;
             for (TopicPartition tp : parts) {
-                log.debug("Seeking to beginning of partition {}", tp);
+                log.info("Seeking to beginning of partition {}", tp);
```

Comment:
I guess we may as well make a similar change in `seekToEnd`?

Reply:
Good point, done.